src/NServiceBus.Core/Transports/TransportDefinition.cs

```diff
@@ -26,6 +26,6 @@ public abstract partial class TransportDefinition
         /// <param name="settings">An instance of the current settings.</param>
         /// <param name="connectionString">The connection string.</param>
         /// <returns>The supported factories.</returns>
-        protected internal abstract TransportInfrastructure Initialize(SettingsHolder settings, string connectionString);
+        public abstract TransportInfrastructure Initialize(SettingsHolder settings, string connectionString);
```

Comment:
this only needs to be public because of the transport tests, do I see that correctly?